guides/get-started/readme.md

```diff
@@ -11,7 +11,7 @@ Using the Stellar network, you can build mobile wallets, banking tools, smart de
 
 The easiest way to install Horizon is by using [**stellar/quickstart** docker image](https://hub.docker.com/r/stellar/quickstart/).
 
-Stellar.org maintains [JavaScript](https://github.com/stellar/js-stellar-sdk), [Java](https://github.com/stellar/java-stellar-sdk), and [Go](https://github.com/stellar/go/tree/master/clients/horizon)-based SDKs for communicating with Horizon. There are also community-maintained SDKs for [Ruby](https://github.com/stellar/ruby-stellar-sdk), [Python](https://github.com/StellarCN/py-stellar-base), and [C#](https://github.com/QuantozTechnology/csharp-stellar-base).
+Stellar.org maintains [JavaScript](https://github.com/stellar/js-stellar-sdk), [Java](https://github.com/stellar/java-stellar-sdk), and [Go](https://github.com/stellar/go/tree/master/clients/horizon)-based SDKs for communicating with Horizon. There are also community-maintained SDKs for [Ruby](https://github.com/stellar/ruby-stellar-sdk), [Python](https://github.com/StellarCN/py-stellar-base), and [C#](https://github.com/elucidsoft/dotnet-stellar-sdk).
```

Comment:
Could you change the last sentence about community-maintained SDKs to the link to the list of SDKs in `reference/readme.md`?

Reply:
Sure will do when i arrive at home!